packages/cmf/src/sagas/http.js

```diff
@@ -111,6 +117,74 @@ export function httpFetch(url, config, method, payload) {
 		.then(handleHttpResponse)
 		.catch(handleError);
 }
+export function cloneError(initialError, resultError) {
+	const errorToEnrich = Object.assign({}, resultError);
+	return initialError.stack.response
+		.clone()
+		.text()
+		.then(response => {
+			try {
+				errorToEnrich.stack.response = response;
+				errorToEnrich.stack.messageObject = JSON.parse(response);
+				return Promise.resolve(errorToEnrich);
+			} catch (exception) {
+				errorToEnrich.stack.messageObject = 'Parsing response error';
+				return Promise.reject(errorToEnrich);
+			}
+		});
+}
+export function* errorProcessing(error, httpAction) {
+	const errorObject = {
+		name: error.name,
+		message: error.description || error.message,
```

Comment:
error.message is the official spec. what is error.description ?

Reply:
https://github.com/Talend/ui/blob/8386cc93a9c42e2de38aa5cd77a27882794a6b13/packages/cmf/src/middlewares/http/middleware.js#L181